persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -75,7 +75,7 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with T
   override def receiveCommand: Receive = {
 
     case BeginValidation =>
-      val ctx = ValidationContext(None)
+      val ctx = ValidationContext(None, None)
```

Comment:
Does anything prevent us from using the actual year from the submission? If not, then let's do it. If so, then that seems like something we need to solve before calling this issue done.

If it can't be done, then it's arguably a fatal flaw in this approach. (In which case, thank you for doing the work to discover it!) Or maybe there's a work-around.
If it can be done, but something is blocking it at the moment, then I vote for hard-coding it here, perhaps with a big blaring comment explaining what's up.
It it can be done now, see above. :)